taxi/views.py

```diff
@@ -52,3 +93,16 @@
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverUpdateView(LoginRequiredMixin, generic.edit.UpdateView):
+    model = Driver
+    fields = "__all__"
+    success_url = reverse_lazy("taxi:driver-list")
+    template_name = "taxi/driver_form.html"
```

Comment:
The same comment as above applies here. You don't need to override `template_name` if it's the same as the default one. The default `template_name` for an UpdateView is `<app name>/<model name>_form.html`. So in this case, you don't need to specify `template_name` as `taxi/driver_form.html` because it's the default one.